CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Changed
 - Technical - Upgrade to babel 7 stable.
+- Smart Field Search - Ease the search on smart fields and deprecate old ways to do it.
```

Comment:
[Thinking] IMO this is not explicit enough. Something like : 
`Ease the search on smart fields by requiring a condition instead of a query, and deprecate old way to do it.` Give a little bit more details